sass/death.sass

```diff
@@ -0,0 +1,46 @@
+#deathAnimation
+  background-image: url(/img/death0.png);
+  background-size: 200px;
+  background-repeat: no-repeat;
+  position: absolute;
+  z-index: 1;
+  left: 600px;
+  top: 350px;
+  width: 250px;
+  height: 250px;
+  overflow: auto;
+  display: none;
+  animation-delay: 4s
+  animation-fill-mode: forwards
+  animation: jumpscare .5s
+  animation-timing-function: linear
+  animation-direction: alternate;
+
+@keyframes jumpscare
```

Comment:
for the jumpescare, instead of a linear approach for the monster, maybe have the scale go from like 1,1;1.5,1.5; 2,2; then 4,4. So he slowly comes then it's like he runs at you?